documentation/cluster-controller/cluster-controller.adoc

```diff
@@ -0,0 +1,267 @@
+= Cluster Controller
+
+The cluster controller is in charge to deploy a Kafka cluster alongside a Zookeeper ensemble. It's also able to deploy a 
```

Comment:
Do we want to keep the _Zookeeper_ hidden here? It is an implementation detail, but _to deploy a Kafka cluster alongside a Zookeeper ensemble_ sounds quite prominent.

Reply:
What do you mean ?